ghost/admin/package.json

```diff
@@ -40,6 +40,7 @@
     "@glimmer/component": "1.1.2",
     "@html-next/vertical-collection": "3.0.0",
     "@sentry/ember": "7.78.0",
+    "@sentry/integrations": "^7.80.0",
```

Comment:
Should we use a pinned version here? i.e `7.80.0`

Reply:
Good catch! Updated :)